Scripts/UnitTests/TestVisualRecognitionRC.cs

```diff
@@ -45,25 +43,25 @@ public class TestVisualRecognitionRC : UnitTest
         private string _classifierID = "";
         private string _imageURL = "https://upload.wikimedia.org/wikipedia/commons/e/e9/Official_portrait_of_Barack_Obama.jpg";
 
-#if DELETE_TRAINED_CLASSIFIER
+#if TRAIN_DELETE_CLASSIFIER
```

Comment:
Whats the reason behind using same variable for both train and delete?

Reply:
Because I don't want to pollute our instance of VisRec with classifiers. If I create a classifier I should delete it after I have tested it works.

Reply:
Makes sense 👍